packages/mobile/src/fees/saga.ts

```diff
@@ -20,9 +20,10 @@ const TAG = 'fees/saga'
 // Prevents us from having to recreate txs and estimate their gas each time
 const feeGasCache = new Map<FeeType, BigNumber>()
 // Just use default values here since it doesn't matter for fee estimation
+
 const placeholderSendTx: BasicTokenTransfer = {
   recipientAddress: CeloDefaultRecipient.address,
-  amount: web3.utils.fromWei('1'),
+  amount: getContractKit().web3.utils.fromWei('1'),
```

Comment:
a getWeb3() helper might be a good idea, seems like we drill down through CK in many places